python/cuml/dask/ensemble/base.py

```diff
@@ -112,6 +117,17 @@ def _fit(self, model, dataset, convert_dtype):
                     workers=[worker],
                     pure=False)
             )
+        if len(self.workers) > len(self.active_workers):
+            if self.ignore_empty_partitions:
+                warn_text = "Data was not split among all workers"\
+                            " using only %d workers to fit" %\

```

Comment:
Please add a note to the warning like: "This will only train %d estimators instead of the requested %d" (self.n_estimators_per_worker * len(self.active_workers), self.n_estimators)

Reply:
Since `self.n_estimators_per_worker` is a list, this will yield the list itself. I've added an updated number of workers assuming they get divided among all workers equally.